nuxeo-mqueues-core/src/main/java/org/nuxeo/ecm/platform/importer/mqueues/mqueues/chronicle/ChronicleMQManager.java

```diff
@@ -39,8 +40,16 @@
     private static final Log log = LogFactory.getLog(ChronicleMQManager.class);
     private final Path basePath;
 
+    private final String retentionPolicy;
+
     public ChronicleMQManager(Path basePath) {
         this.basePath = basePath;
+        this.retentionPolicy = DEFAULT_RETENTION_POLICY;
+    }
+
+    public ChronicleMQManager(Path basePath, String retentionPolicy) {
+        this.basePath = basePath;
```

Comment:
to be more precise here you pass the retention duration not a retention policy,
we could imagine retention policy that also take care of a number of file/total size and a duration
so first impl is to have a default retention policy with duration and we just pass the retentionDuration
with some javadoc to descibe the accepted format